turbo.json

```diff
@@ -7,16 +7,6 @@
       "dependsOn": ["^transpile", "^typecheck"]
     },
 
-    "dev": {
```

Comment:
What are these changes for?

Reply:
I realized I left some old dev mode stuff around.